src/drawer.canvas.js

```diff
@@ -48,6 +48,25 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.Canvas, {
 
         this.clearWave();
     },
+    
+    updateWidthnHeight: function () {
+        var width = Math.round(this.width / this.params.pixelRatio);
+        var height = this.height;
+
+        this.waveCc.canvas.width = this.width;
+        this.waveCc.canvas.height = this.height;
+        this.style(this.waveCc.canvas, { width: width + 'px'});
+        this.style(this.waveCc.canvas, { height: height + 'px'});
```

Comment:
this.style(this.waveCc.canvas, {
    width: width + 'px',
    height: height + 'px'
});